common/protob/messages-management.proto

```diff
@@ -131,6 +131,24 @@ message ChangePin {
     optional bool remove = 1;   // is PIN removal requested?
 }
 
+/**
+ * Request: Starts workflow for enabling/regenerating/disabling SD card salt
+ * @start
+ * @next Success
+ * @next Failure
+ */
+message SdSalt {
+    optional SdSaltOperationType operation = 1;
+    /**
+    * Structure representing SD card salt operation
+    */
+    enum SdSaltOperationType {
+        ENABLE = 0;
```

Comment:
This is just a nitpick. but I'd say `DISABLE = 0` and `ENABLE = 1` make more sense.

Reply:
ACK